### ISSUE TITLE
CME crash during launch

### ISSUE THREAD

**Issue by embeddedt** [SlimeKnights/TinkersConstruct#5123]
### Minecraft Version

1.18.2

### Forge Version

40.2.0

### Mantle Version

1.9.43

### Tinkers' Construct Version

3.6.3.111

### Describe your issue

During the client setup phase a CME is thrown by the JSON Things plugin:

```
[11:47:35] [Worker-Main-4/ERROR] [ne.mi.fm.ja.FMLModContainer/LOADING]: Caught exception during event FMLClientSetupEvent dispatch for modid tconstruct
java.util.ConcurrentModificationException: null
	at java.util.HashMap.computeIfAbsent(HashMap.java:1221) ~[?:?] {re:mixin}
	at net.minecraft.client.renderer.item.ItemProperties.register(ItemProperties.java:60) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at slimeknights.tconstruct.library.client.model.TinkerItemProperties.registerToolProperties(TinkerItemProperties.java:83) ~[TConstruct-1.18.2-3.6.3.111.jar%2386!/:1.18.2-3.6.3.111] {re:classloading}
	at slimeknights.tconstruct.plugin.jsonthings.PluginClient.clientSetup(PluginClient.java:20) ~[TConstruct-1.18.2-3.6.3.111.jar%2386!/:1.18.2-3.6.3.111] {re:classloading}
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:247) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:239) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:106) ~[javafmllanguage-1.18.2-40.2.0.jar%2389!/:?] {}
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:107) ~[fmlcore-1.18.2-40.2.0.jar%2388!/:?] {re:mixin}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) [?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) [?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?] {re:mixin,re:computing_frames}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?] {re:mixin,re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?] {re:mixin}
```

I believe the fix is just to move the call to `registerToolProperties` inside `event.enqueueWork` so that this runs on the main thread.

### Other mods

Tinkers Netherflat Challenge

### Tried reproducing with just Tinkers?

No

### Performance Enchancers

Other (specify under other mods)

### Searched for known issues?

Searched open issues

**Comment by KnightMiner** [comment]
Give the full crash report/log, not the single line.

**Comment by embeddedt** [comment]
```
---- Minecraft Crash Report ----
// I blame Dinnerbone.

Time: 4/4/23, 11:47 AM
Description: Mod loading error has occurred

java.lang.Exception: Mod Loading has failed
	at net.minecraftforge.logging.CrashReportExtender.dumpModLoadingCrashReport(CrashReportExtender.java:55) ~[forge-1.18.2-40.2.0-universal.jar%2392!/:?] {re:classloading}
	at net.minecraftforge.client.loading.ClientModLoader.completeModLoading(ClientModLoader.java:170) ~[forge-1.18.2-40.2.0-universal.jar%2392!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.lambda$new$1(Minecraft.java:557) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:modernfix.mixins.json:core.MinecraftMixin,pl:mixin:APP:modernfix.mixins.json:feature.measure_time.MinecraftMixin,pl:mixin:APP:modernfix.mixins.json:bugfix.concurrency.MinecraftMixin,pl:mixin:APP:freecam.mixins.json:MinecraftMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.Util.m_137521_(Util.java:397) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:mixin,re:classloading,pl:mixin:APP:modernfix.mixins.json:perf.thread_priorities.UtilMixin,pl:mixin:A}
	at net.minecraft.client.Minecraft.lambda$new$2(Minecraft.java:551) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:modernfix.mixins.json:core.MinecraftMixin,pl:mixin:APP:modernfix.mixins.json:feature.measure_time.MinecraftMixin,pl:mixin:APP:modernfix.mixins.json:bugfix.concurrency.MinecraftMixin,pl:mixin:APP:freecam.mixins.json:MinecraftMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.screens.LoadingOverlay.m_6305_(LoadingOverlay.java:135) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.m_109093_(GameRenderer.java:879) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1046) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:modernfix.mixins.json:core.MinecraftMixin,pl:mixin:APP:modernfix.mixins.json:feature.measure_time.MinecraftMixin,pl:mixin:APP:modernfix.mixins.json:bugfix.concurrency.MinecraftMixin,pl:mixin:APP:freecam.mixins.json:MinecraftMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:665) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:rubidium.mixins.json:core.MixinMinecraftClient,pl:mixin:APP:flywheel.mixins.json:PausedPartialTickAccessor,pl:mixin:APP:modernfix.mixins.json:core.MinecraftMixin,pl:mixin:APP:modernfix.mixins.json:feature.measure_time.MinecraftMixin,pl:mixin:APP:modernfix.mixins.json:bugfix.concurrency.MinecraftMixin,pl:mixin:APP:freecam.mixins.json:MinecraftMixin,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:APP:create.mixins.json:WindowResizeMixin,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:classloading,re:mixin,pl:runtimedistcleaner:A,pl:mixin:A,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.2.0.jar%2351!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%2339!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:57) [ForgeWrapper-mmc2.jar:mmc2] {}
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88) [NewLaunch.jar:?] {}
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126) [NewLaunch.jar:?] {}
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71) [NewLaunch.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at java.util.HashMap.computeIfAbsent(HashMap.java:1221) ~[?:?] {re:mixin}
-- MOD tconstruct --
Details:
	Mod File: <home>/.local/share/PrismLauncher/instances/Tinkers' NetherFlat Challenge/minecraft/mods/TConstruct-1.18.2-3.6.3.111.jar
	Failure message: Tinkers' Construct (tconstruct) encountered an error during the sided_setup event phase
		java.util.ConcurrentModificationException: null
	Mod Version: 3.6.3.111
	Mod Issue URL: https://github.com/SlimeKnights/TinkersConstruct/issues
	Exception message: java.util.ConcurrentModificationException
Stacktrace:
	at java.util.HashMap.computeIfAbsent(HashMap.java:1221) ~[?:?] {re:mixin}
	at net.minecraft.client.renderer.item.ItemProperties.register(ItemProperties.java:60) ~[client-1.18.2-20220404.173914-srg.jar%2387!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at slimeknights.tconstruct.library.client.model.TinkerItemProperties.registerToolProperties(TinkerItemProperties.java:83) ~[TConstruct-1.18.2-3.6.3.111.jar%2386!/:1.18.2-3.6.3.111] {re:classloading}
	at slimeknights.tconstruct.plugin.jsonthings.PluginClient.clientSetup(PluginClient.java:20) ~[TConstruct-1.18.2-3.6.3.111.jar%2386!/:1.18.2-3.6.3.111] {re:classloading}
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:247) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:239) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-5.0.3.jar%2336!/:?] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:106) ~[javafmllanguage-1.18.2-40.2.0.jar%2389!/:?] {}
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:107) ~[fmlcore-1.18.2-40.2.0.jar%2388!/:?] {re:mixin}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804) ~[?:?] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[?:?] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?] {}
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?] {re:mixin,re:computing_frames}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?] {re:mixin,re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?] {re:mixin}


-- System Details --
Details:
	Minecraft Version: 1.18.2
	Minecraft Version ID: 1.18.2
	Operating System: Linux (amd64) version 6.2.8-arch1-1
	Java Version: 17.0.6, N/A
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 2217970688 bytes (2115 MiB) / 3221225472 bytes (3072 MiB) up to 3221225472 bytes (3072 MiB)
	CPUs: 8
	Processor Vendor: GenuineIntel
	Processor Name: 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
	Identifier: Intel64 Family 6 Model 140 Stepping 1
	Microarchitecture: Tiger Lake
	Frequency (GHz): 2.40
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: TigerLake-LP GT2 [Iris Xe Graphics]
	Graphics card #0 vendor: Intel Corporation (0x8086)
	Graphics card #0 VRAM (MB): 256.00
	Graphics card #0 deviceId: 0x9a49
	Graphics card #0 versionInfo: unknown
	Virtual memory max (MB): 20245.86
	Virtual memory used (MB): 9333.37
	Swap memory total (MB): 16384.00
	Swap memory used (MB): 2140.25
	JVM Flags: 18 total; -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=100 -XX:+UnlockExperimentalVMOptions -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=20 -XX:G1MaxNewSizePercent=30 -XX:G1HeapRegionSize=32M -XX:G1ReservePercent=30 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:MaxTenuringThreshold=1 -Xms3072m -Xmx3072m
	ModLauncher: 9.1.3+9.1.3+main.9b69c82a
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services:
		 mixin PLUGINSERVICE
		 eventbus PLUGINSERVICE
		 slf4jfixer PLUGINSERVICE
		 object_holder_definalize PLUGINSERVICE
		 runtime_enum_extender PLUGINSERVICE
		 capability_token_subclass PLUGINSERVICE
		 accesstransformer PLUGINSERVICE
		 runtimedistcleaner PLUGINSERVICE
		 mixin TRANSFORMATIONSERVICE
		 fml TRANSFORMATIONSERVICE
	FML Language Providers:
		minecraft@1.0
		lowcodefml@null
		javafml@null
	Mod List:
		client-1.18.2-20220404.173914-srg.jar             |Minecraft                     |minecraft                     |1.18.2              |ENQUEUE_IM|Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		TConstruct-1.18.2-3.6.3.111.jar                   |Tinkers' Construct            |tconstruct                    |3.6.3.111           |ERROR     |Manifest: NOSIGNATURE
		BorderlessWindow-1.18-1.4.0.jar                   |Borderless Window             |borderlesswindow              |1.18-1.4.0          |ENQUEUE_IM|Manifest: NOSIGNATURE
		MouseTweaks-forge-mc1.18-2.21.jar                 |Mouse Tweaks                  |mousetweaks                   |2.21                |ENQUEUE_IM|Manifest: NOSIGNATURE
		Jade-1.18.2-forge-5.2.6.jar                       |Jade                          |jade                          |5.2.6               |ENQUEUE_IM|Manifest: NOSIGNATURE
		jei-1.18.2-9.7.1.255.jar                          |Just Enough Items             |jei                           |9.7.1.255           |ENQUEUE_IM|Manifest: NOSIGNATURE
		CustomFluidMixin-1.2.1-mc1.18.2.jar               |Custom Fluid Mixin            |customfluidmixin              |1.2.1-mc1.18.2      |ENQUEUE_IM|Manifest: NOSIGNATURE
		IguanaTweaksReborn-2.12.11-mc1.18.2.jar           |Iguana Tweaks Reborn          |iguanatweaksreborn            |2.12.11-mc1.18.2    |ENQUEUE_IM|Manifest: NOSIGNATURE
		rubidium-0.5.4.jar                                |Rubidium                      |rubidium                      |0.5.4               |ENQUEUE_IM|Manifest: NOSIGNATURE
		InsaneLib-1.5.3-mc1.18.2.jar                      |InsaneLib                     |insanelib                     |1.5.3               |ENQUEUE_IM|Manifest: NOSIGNATURE
		flywheel-forge-1.18.2-0.6.8.a.jar                 |Flywheel                      |flywheel                      |0.6.8.a             |ENQUEUE_IM|Manifest: NOSIGNATURE
		Mantle-1.18.2-1.9.43.jar                          |Mantle                        |mantle                        |1.9.43              |ENQUEUE_IM|Manifest: NOSIGNATURE
		create-1.18.2-0.5.0.i.jar                         |Create                        |create                        |0.5.0.i             |ENQUEUE_IM|Manifest: NOSIGNATURE
		shutupexperimentalsettings-1.0.5.jar              |Shutup Experimental Settings! |shutupexperimentalsettings    |1.0.5               |ENQUEUE_IM|Manifest: NOSIGNATURE
		modernfix-mc1.18.2-2.2.2.jar                      |ModernFix                     |modernfix                     |2.2.2               |ENQUEUE_IM|Manifest: NOSIGNATURE
		AutoRegLib-1.7-53.jar                             |AutoRegLib                    |autoreglib                    |1.7-53              |ENQUEUE_IM|Manifest: NOSIGNATURE
		configured-2.0.0-1.18.2.jar                       |Configured                    |configured                    |2.0.0               |ENQUEUE_IM|Manifest: NOSIGNATURE
		freecam-forge-1.1.8+1.18.jar                      |Freecam                       |freecam                       |1.1.8+1.18          |ENQUEUE_IM|Manifest: NOSIGNATURE
		clear-skies-forge-mc118-2.0.92.jar                |Clear Skies                   |clear_skies                   |2.0.92              |ENQUEUE_IM|Manifest: NOSIGNATURE
		lazydfu-1.0-1.18+.jar                             |LazyDFU                       |lazydfu                       |0.1.3               |ENQUEUE_IM|Manifest: NOSIGNATURE
		ProgressiveBosses-3.6.5-mc1.18.2.jar              |Progressive Bosses            |progressivebosses             |3.6.5               |ENQUEUE_IM|Manifest: NOSIGNATURE
		architectury-4.11.89-forge.jar                    |Architectury                  |architectury                  |4.11.89             |ENQUEUE_IM|Manifest: NOSIGNATURE
		catalogue-1.6.2-1.18.2.jar                        |Catalogue                     |catalogue                     |1.6.2               |ENQUEUE_IM|Manifest: NOSIGNATURE
		light-overlay-6.0.5-forge.jar                     |Light Overlay                 |lightoverlay                  |6.0.5               |ENQUEUE_IM|Manifest: NOSIGNATURE
		JEITweaker-1.18.2-3.0.0.9.jar                     |JEI Tweaker                   |jeitweaker                    |3.0.0.9             |ENQUEUE_IM|Manifest: NOSIGNATURE
		JustEnoughResources-1.18.2-0.14.1.171.jar         |Just Enough Resources         |jeresources                   |0.14.1.171          |ENQUEUE_IM|Manifest: NOSIGNATURE
		CraftTweaker-forge-1.18.2-9.1.205.jar             |CraftTweaker                  |crafttweaker                  |9.1.205             |ENQUEUE_IM|Manifest: NOSIGNATURE
		cloth-config-6.4.90-forge.jar                     |Cloth Config v4 API           |cloth_config                  |6.4.90              |ENQUEUE_IM|Manifest: NOSIGNATURE
		forge-1.18.2-40.2.0-universal.jar                 |Forge                         |forge                         |40.2.0              |ENQUEUE_IM|Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		Shields+-1.2.5-mc1.18.2.jar                       |ShieldsPlus                   |shieldsplus                   |1.2.5-mc1.18.2      |ENQUEUE_IM|Manifest: NOSIGNATURE
		goprone-1.18.2-1.7.0.jar                          |GoProne                       |goprone                       |1.18.2-1.7.0        |ENQUEUE_IM|Manifest: NOSIGNATURE
		Quark-3.2-358.jar                                 |Quark                         |quark                         |3.2-358             |ENQUEUE_IM|Manifest: NOSIGNATURE
		cofh_core-1.18.2-9.2.1.39.jar                     |CoFH Core                     |cofh_core                     |9.2.1               |ENQUEUE_IM|Manifest: 75:0b:cc:9b:64:2e:9b:c4:41:d1:95:00:71:ee:87:1a:b3:5e:4b:da:8e:e8:39:00:fd:5d:e5:9c:40:42:33:09
		archers_paradox-1.18.2-3.2.0.12.jar               |Archer's Paradox              |archers_paradox               |3.2.0.12            |ENQUEUE_IM|Manifest: 75:0b:cc:9b:64:2e:9b:c4:41:d1:95:00:71:ee:87:1a:b3:5e:4b:da:8e:e8:39:00:fd:5d:e5:9c:40:42:33:09
		JsonThings-1.18.2-0.4.8.jar                       |Json Things                   |jsonthings                    |0.4.8               |ENQUEUE_IM|Manifest: NOSIGNATURE
		BetterAdvancements-1.18.2-0.2.0.146.jar           |Better Advancements           |betteradvancements            |0.2.0.146           |ENQUEUE_IM|Manifest: NOSIGNATURE
	Flywheel Backend: GL33 Instanced Arrays
	Crash Report UUID: 7af8ae37-ad0e-4157-a4d3-00e5dfe367f6
	FML: 40.2
	Forge: net.minecraftforge:40.2.0
```

**Comment by KnightMiner** [comment]
Fixed in 3.6.4.113
